pages/docs/concepts/asyncapi-document/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Introduction'
+weight: 1
+---
+
+The AsyncAPI document is a YAML or JSON file that describes your implementation of the AsyncAPI specification. Providing a standardized way to document and describe asynchronous and event-driven systems, the AsyncAPI document generates documentation/code and defines the different components of your event-driven application, such as the events, channels, and message formats.
+
+Primarily, it provides a method to produce comprehensive, consistent, and standardized AsyncAPI documentation. Additionally, it serves as a contract for communication between consumers and producers in an event-driven system. It specifies what should be included in the payload when a service produces a message and provides information to the consumer about the properties in the message. 
+
+```YAML
+asyncapi: 3.0.0
+info:
+  title: Cool Example
+  version: 0.1.0
+channels:
+  user/signedup:
+    subscribe:
+      message:
+        description: An event describing that a user just signed up.
+        payload:
+          type: object
+          additionalProperties: false
+          properties:
+            fullName:
+              type: string
+            email:
+              type: string
+              format: email
+            age:
+              type: integer
+              minimum: 18
+```
+
+## AsyncAPI document components
```

Comment:
We refer to these as `Resources` or `Objects` as far as I can see in the spec:

> AsyncAPI Object
This is the root document object for the API specification. It combines resource listing and API declaration together into one document. 
>
> Source: https://www.asyncapi.com/docs/reference/specification/v3.0.0-next-major-spec.12#schema

I say that because `components` is also a thing in the AsyncAPI document: https://www.asyncapi.com/docs/reference/specification/v3.0.0-next-major-spec.12#componentsObject and can sound confusing.

Reply:
Ah, thank you for calling that out!